csrc/host_ir/executor.cpp

```diff
@@ -325,24 +325,38 @@ void HostIrEvaluator::handle(Synchronize* synchronize) {
 }
 
 void HostIrEvaluator::handle(LaunchKernel* launch_kernel) {
-  KernelArgumentHolder args;
+  KernelArgumentHolder args, outputs;
```

Comment:
```suggestion
  KernelArgumentHolder args;
```
I'd move outputs to line 334. In general, try to declare variables close to where they are used.